tower.yml

```diff
@@ -1,5 +1,135 @@
 reports:
   multiqc_report.html:
     display: "MultiQC HTML report"
-  samplesheet.csv:
```

Comment:
@jfy133 I guess I should keep the `samplesheet.csv` and add `database.csv` info here, right?

Reply:
No, `samplesheet.csv` is the input.

Reply:
then yeah, that's exactly what you'd want to do